### ISSUE TITLE
Tray Icon Dissapears

### ISSUE THREAD

**Issue by jonathanmwood** [ChrisAnd1998/TaskbarX#889]
Duplicate of #678.

Description lifted from original issue by theFireProject.

> Short description

Multiple tray icons disappear after left clicking the TaskbarX tray icon. This problem persists after restarting TaskbarX.
Steps to reproduce

Steps to reproduce the bug:

    Start up the computer
    TaskbarX starts automatically
    Left click the TaskbarX tray icon

Steps to restore the taskbar and system tray to its original state (without TaskbarX):

    Stop TaskbarX
    Restart Windows Explorer

The system tray seems to be broken once I left click the TaskbarX tray icon. It disappears together with all other icons after the TaskbarX tray icon. Only one empty icon remains. In the screenshot, you cannot see the mouse pointer but I'm pointing at this empty tray icon. It doesn't have an icon (small image), it doesn't have a tooltip and it doesn't do anything when it is clicked:

If you stop TaskbarX, the empty tray icon will disappear, but the other tray icons will still be missing. To fix this problem, you have to restart Windows Explorer. All missing tray icons will reappear.

But as soon as you start TaskbarX again, the same icons as before will disappear once more and the empty icon will be there again, too.

It is possible to fix the empty tray icon and display the TaskbarX tray icon correctly again. But this won't fix the other missing tray icons. To do that, you have to disable the TaskbarX tray icon in the Configurator, click Apply, enable the tray icon and click Apply once more. Now the empty tray icon is gone and the TaskbarX tray icon is displayed and working, but the other tray icons that disappeared before are still missing.

TaskbarX Version 1.7.6.0
OS: Windows 10 Pro
Version: 21H2

**Comment by SirGouki** [comment]
I am having the same problems on Windows 10 Home Version 21H1.  I only recently determined that TaskbarX was causing the problem, and its been happening since at least 1.7.3

A few other notes: this error force closes the AMD Radeon Catalyst software, and some other software, which must be manually restarted after using the task manager to restart Windows Explorer.  Restarting Windows Explorer after this bug starts also force closes TaskbarX the majority of the time.  This bug can go so far as to hide folders added to the task bar as a toolbar, as well as the clock and notification area.  These things do not happen immediately, but happen over time as the program continues to run.  I have not tested if a difference occurs by switching from MSAA to the other option.

**Comment by eats7** [comment]
Yup I get this issue as well when switching from my tv to my monitor. The clock, notifications etc just disappear.
